python/lsst/ts/mtdome/mtdome_csc.py

```diff
@@ -439,6 +456,7 @@ async def handle_summary_state(self) -> None:
                 await self.connect()
         else:
             await self.disconnect()
+            self.latest_moveAz_command = MoveAzCommandData()
```

Comment:
What happens if disconnect fails? Does the CSC go to FAULT or remain connected and enabled or...? That affects whether this line should appear before disconnecting or after. 

I simplest/clearest option may be to move this line to right before self.connect() above. Then it should be reliably run, regardless of what happens if disconnect fails.